examples/h2o-classification/h2o-example.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import local bentoml repository\n",
+    "import sys\n",
+    "import os\n",
+    "sys.path.insert(0, os.path.join(os.getcwd(), \"..\", \"..\"))\n",
+    "import bentoml\n",
+    "\n",
+    "print(bentoml.__file__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# H2O Classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !curl -i https://raw.githubusercontent.com/multicode/h2o-notebook/master/prostate.csv -o prostate.csv"
```

Comment:
would be great to use bash's if statement here to make this notebook "run-all-able", similar to what we did with the sklearn example

Reply:
done, no longer apply